### ISSUE TITLE
Timeline for 1.0

### ISSUE THREAD

**Issue by jparise** [Microsoft/thrifty#258]
Has the roadmap for 1.0 been defined yet?

I'd like to see a 1.0-RC3 with the most recent changes. In particular, it would be nice to bump our codebase with #236, #241, and #250 because they're quality-of-line improvements.

It seems like #257 might be a blocking issue for a full 1.0 release though.

**Comment by benjamin-bader** [comment]
You're right about #257.  I'm testing a fix now.  Once that's in, I think we're ready for 1.0.
